ds/ds.php

```diff
@@ -842,6 +842,16 @@ public function jsonSerialize()
 
     class Deque implements Sequence
     {
+        /**
+         * Creates a new instance, using either a traversable object or an array for the initial values.
+         * @param mixed ...$values A traversable object or an array to use for the initial values.
+         *
+         * @link https://www.php.net/manual/en/ds-map.construct.php
```

Comment:
Seems this link is for Map class but not Deque. And correct link for Deque class is https://www.php.net/manual/en/ds-deque.construct.php